perf-measures/bundle-check/scripts/check-bundle-size.ts

```diff
@@ -19,12 +19,21 @@ async function main() {
 
     const bundleSize = fs.statSync(tempFilePath).size
     const metrics = []
+
     metrics.push({
-      key: 'bundle-size',
-      name: 'Bundle Size',
+      key: 'bundle-size-b',
+      name: 'Bundle Size (B)',
       value: bundleSize,
       unit: 'B',
     })
+
+    metrics.push({
+      key: 'bundle-size-kb',
+      name: 'Bundle Size (KB)',
+      value: (bundleSize / 1024).toFixed(2),
```

Comment:
I don't know, as I have never tried it, but I was curious about the following warning message.

https://github.com/honojs/hono/actions/runs/11945102648/job/33297187610?pr=3696#step:7:94
`Skip collecting custom metrics: json: float unexpected end of JSON input`

What if we set the value as a string?

Reply:
This is the only issue that has the same error, not octocov.  
So this error itself is unusual...  
Hmmm...  

https://github.com/pluja/whishper/issues/78